cache/manager_test.go

```diff
@@ -1266,10 +1266,6 @@ func TestSharingCompressionVariant(t *testing.T) {
 	require.NoError(t, err)
 	defer cleanup()
 
-	ctx, done, err := leaseutil.WithLease(ctx, co.lm, leaseutil.MakeTemporary)
```

Comment:
Why is this being removed?

Reply:
To check if the lease on each ref (not the one in `ctx`) is effective. The context lease prevents contents are garbage collected during its lifecycle so hides the issue.